Firmware/Marlin_main.cpp

```diff
@@ -6641,6 +6641,44 @@ void process_commands()
     break;
     #ifdef PREVENT_DANGEROUS_EXTRUDE
 
+    /*!
+    ### M256 - Set LCD brightness <a href="https://reprap.org/wiki/G-code#M256:_Set_LCD_brightness">M256: Set LCD brightness</a>
+    Set and/or get the LCD brightness. The value is constrained based on the LCD, but typically a value of 0 is the dimmest and 255 is the brightest.
+    #### Usage
+
+        M256 [ B | D | S | T ]
+
+    #### Parameters
+    - `B` - Normal Brightness value (0 - 255), default 130
+    - `D` - Dimmed Brightness value (0 - 255), default 50
+    - `S` - Brightness mode, default Auto
+      - `0` - Dim
+      - `1` - Bright
+      - `2` - Auto
+    - `T` - Brightness timeout  (15 - 900), default 15 seconds
+    */
+    #ifdef LCD_BL_PIN
+     case 256:
+    {
+      if (backlightSupport) {
+        if (code_seen('B') ) backlightLevel_HIGH=code_value_uint8();
+        if (code_seen('D')) backlightLevel_LOW=code_value_uint8();
+        if (code_seen('S')) {
+          backlightMode=code_value_uint8();
```

Comment:
```suggestion
          backlightMode = max(code_value_uint8(), BACKLIGHT_MODE_AUTO);
```

Reply:
Thanks after getting
```
warning: enumeral and non-enumeral type in conditional expression [-Wextra]
 #define max(a,b) ((a)>(b)?(a):(b))
                   ~~~~~~~^~~~~~~~
```
changed it to `if (code_seen('S')) backlightMode = max(static_cast<Backlight_Mode>(code_value_uint8()), BACKLIGHT_MODE_AUTO);` and the warning is gone.